src/main/java/Lotto/OperatorType.java

```diff
@@ -0,0 +1,18 @@
+package lotto;
+
+public enum OperatorType {
+    ADD("+"),
```

Comment:
아래와 같은 형식으로 연산자가 직접 계산하도록 구현해 보시면 좋은 연습이 되실 것 같아요 😃 
```suggestion
    ADD("+", (x, y) -> x + y),
```

Reply:
신세계..! java8부터는 함수를 직접 넣을수 있어서 유용하네요. 👍🏻